<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alteruser_Dialog</class>
 <widget class="QDialog" name="alteruser_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改用户</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="passWord_lineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>确认密码：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="passWord2lineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>权限：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="authority_comboBox">
      <item>
       <property name="text">
        <string>管理员</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>普通用户</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>备注：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="note_lineEdit"/>
    </item>
    <item row="5" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ok_pushButton">
        <property name="text">
         <string>确定</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image/update.png</normaloff>image/update.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_pushButton">
        <property name="text">
         <string>取消</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image/delete.png</normaloff>image/delete.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="userName_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
